tests/cupyx_tests/scipy_tests/stats_tests/test_morestats.py

```diff
@@ -0,0 +1,109 @@
+import cupy
+
+from cupy import testing
+
+import cupyx
+import cupyx.scipy.stats  # NOQA
+
+import scipy.stats  # NOQA
+
+
+atol = {
+    'default': 1e-6,
+    cupy.float16: 5e-2,
+    cupy.float64: 1e-14
+}
+rtol = {
+    'default': 1e-6,
+    cupy.float16: 5e-2,
+    cupy.float64: 1e-14
+}
+
+
+atol_low = {
+    'default': 1e-6,
+    cupy.float16: 5e-3,
+    cupy.float32: 1e-5,
+    cupy.float64: 5e-3
+}
+rtol_low = {
+    'default': 1e-6,
+    cupy.float16: 5e-3,
+    cupy.float32: 1e-5,
+    cupy.float64: 5e-3
+}
+
+
+@testing.with_requires('scipy')
+class TestBoxcox_llf:
+
+    @testing.for_all_dtypes()
+    @testing.numpy_cupy_allclose(
+        scipy_name='scp',
+        atol=atol_low,
+        rtol=rtol_low
+    )
+    def test_array_1dim(self, xp, scp, dtype):
+        data = testing.shaped_arange((2,), xp, dtype=dtype)
+        lmb = 4.0
+        return scp.stats.boxcox_llf(lmb, data)
```

Comment:
I tried to use `testing.shaped_random()` here, but the test cases failed.

A short summary about it:
1. The first thing that immediately clicked in my mind was relaxing tolerance. I tried as lenient tolerance as possible but got I got such errors:
<details>
<summary>Error Log 1</summary>

<p>

```python
E       AssertionError: 
E       Not equal to tolerance rtol=0.0001, atol=0.0001
E       
E       Mismatched elements: 1 / 1 (100%)
E       Max absolute difference: 0.00101796
E       Max relative difference: 0.00123653
E        x: array(0.822224)
E        y: array(0.823242)
```

</p>
</details>
2. Then at one possible value, I got:
<details>
<summary>Error Log 2</summary>

<p>

```python
E       AssertionError: 
E       Not equal to tolerance rtol=0.005, atol=0.005
E       
E       x and y -inf location mismatch:
E        x: array(0.280961)
E        y: array(-inf)
```

</p>
</details>
3. I than thought to check was SciPy returns (as we did in `log_softmax`) And it worked completely fine. Please see below:
<details>
<summary>Code Snippet</summary>

<p>

```python
>>> import cupy as cp
>>> from cupyx.scipy.stats import boxcox_llf
>>> lmb = 1.0
>>> boxcox_llf(lmb, x)
array(inf)

>>> from scipy.stats import boxcox_llf
>>> import numpy as np
>>> x = np.array([2], dtype=np.float32)
>>> boxcox_llf(lmb, x)
inf
```


</p>
</details>

Reply:
`shaped_range`'s scale defaults to `10`, so I guess it makes overflow on float16. Please try smaller values.

https://github.com/cupy/cupy/blob/3fedbb957b79782089868540065ca24b088f29c6/cupy/testing/_helper.py#L129

Reply:
This works! Will updated it. Thanks!